File: src/Ebay/Server.Application.New/Abstractions/Repositories/ITubeWorkingPointsRepository.fs
namespace Server.Application.New.Abstractions.Repositories

open System
open Server.Application.New.Abstractions
open Server.Domain.Measurements

/// <summary>
/// Репозиторий агрегата рабочей точки лампы.
/// </summary>
type ITubeWorkingPointsRepository =
    inherit IRepository<TubeWorkingPoint, Guid>

File: src/Ebay/Server.Application.Abstractions.Driven/Abstractions/Repositories/ITubeWorkingPointsRepository.cs
using Server.Application.Abstractions.Driven.Abstractions.Abstractions;
using Server.Domain.Measurements;

namespace Server.Application.Abstractions.Driven.Abstractions.Repositories;

/// <summary>
/// Репозиторий агрегата рабочей точки лампы.
/// </summary>
public interface ITubeWorkingPointsRepository : IRepository<TubeWorkingPoint, Guid>
{

}
